dev.skynomads.Seabird.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: dev.skynomads.Seabird
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: seabird
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
why do you need this? Gtk4 does not need it.

Reply:
Not sure, saw it somewhere and I assumed it was required for notifications. Not that there are notifications anyway. I removed it